### PR TITLE
tpcc: use lookup join in stock level

### PR DESCRIPTION
**Pull request by jordanlewis**

### CODE REVIEW

**Review by rjnn — APPROVED** [review]
LGTM

File: tpcc/stock_level.go [hunk 1]
```diff
@@ -57,6 +57,12 @@ func (s stockLevel) run(db *sql.DB, wID int) (interface{}, error) {
 		dID:       rand.Intn(9) + 1,
 	}
 
+	// This is the only join in the application, so we don't need to worry about
+	// this setting persisting incorrectly across queries.
+	if _, err := db.Exec(`set experimental_force_lookup_join=true`); err != nil {
```
Comment (petermattis) [inline tpcc/stock_level.go:62]: This is not doing what you think it is doing. This is setting a session-level option on a single connection and we need to set this option on every connection. I think you need to move this inside the `crdb.ExecuteTx` and use `tx.Exec` instead.

Comment (rjnn) [reply tpcc/stock_level.go:62]: But this is being called inside `stockLevel.run`. It's not being done transactionally, but it is being done every time before the join is run. Am I missing something to do with the multiplexing of connections internally in `db`?

Comment (petermattis) [reply tpcc/stock_level.go:62]: There is no guarantee that the connection used for the `db.Exec` is the same one used for the transaction below.

**Pull request merged**